PJ05/src/MessageServerWorker.java

```diff
@@ -206,20 +208,15 @@ CreateConversationResponse process(CreateConversationRequest createConversationR
             InvalidConversationNameException, UserNotFoundException {
         //群主
         UUID createConversation_uuid = createConversationRequest.uuid;
-        //群员ID
-        UUID[] particpant_uuid = createConversationRequest.user_uuids;
+
 
         String name = createConversationRequest.name;
 
         if (currentUser == null) {
             throw new NotLoggedInException();
-        } else if (particpant_uuid.length == 0) {
-            //有个问题(GOOGLE DOC)
-            //
-            throw new UserNotFoundException();
         } else {
-            Conversation conversation = system.createConversation(name, createConversation_uuid, particpant_uuid);
-            return new CreateConversationResponse(true, "", createConversation_uuid, particpant_uuid);
+            Conversation conversation = system.createConversation(name, createConversation_uuid);
+            return new CreateConversationResponse(true, "", createConversation_uuid, createConversation_uuid);
```

Comment:
The request includes `conversation_name` and `user_uuids`, while the response includes `conversation_uuid`